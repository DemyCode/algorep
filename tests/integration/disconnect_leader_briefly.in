1
3
no
timeout 2
sleep 350
entry 1 test
start 1
wait_finish 1
sleep 250
print_local_log 2
print_local_log 3
print_local_log 4
crash 2
sleep 90
recover 2
sleep 200
get_state 2
entry 1 test2
wait_finish 1
sleep 250
print_local_log 2
print_local_log 3
print_local_log 4
stop 1
stop 2
stop 3
stop 4