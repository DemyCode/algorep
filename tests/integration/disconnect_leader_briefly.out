2 [1, "test"]
3 [1, "test"]
4 [1, "test"]
State 2: LEADER
2 [1, "test"] [1, "test2"]
3 [1, "test"] [1, "test2"]
4 [1, "test"] [1, "test2"]
